ui/src/pages/Admin/Edit/UploadStory.tsx

```diff
@@ -221,6 +221,7 @@ export const UploadStory: React.FC<StoryProps> = ({
   const history = useHistory();
 
   const startYear = new Date().getFullYear();
+  // const prefilledYear = id ? currentStory.year : startYear.toString();
```

Comment:
Can we 🔥?

Reply:
"You can ignore 75% of my PR comments" -- you Tuesday night